backend/storage-module/services/serviceStorage.ts

```
@@ -49,8 +59,25 @@ export class StorageService {
+   *
+   * @param filePath - Path to the file in Firebase Storage (can be a URL)
+   * @returns - void
+   * @throws - Error if file path is not provided
+   */
+  static async deleteFile(filePath: string) {
+    if (!filePath) {
+      throw new Error("File path is required.");
+    }
+
+    if (filePath.includes("https://firebasestorage.googleapis.com")) {
+      filePath = await this.extractFilePathFromUrl(filePath);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)